scripts/chief_deploy.py

```diff
@@ -71,7 +71,7 @@ def prime_app(ctx):
 @hostgroups(settings.CELERY_HOSTGROUP, remote_kwargs={'ssh_key': settings.SSH_KEY})
 def update_celery(ctx):
     ctx.remote(settings.REMOTE_UPDATE_SCRIPT)
-    ctx.remote('/usr/bin/supervisorctl restart %s' % settings.CELERY_SERVICE)
+    ctx.remote('/usr/bin/supervisorctl mrestart celery*')
```

Comment:
no typo, we use a supervisor plugin that adds wildcard support via an own command